Gulpfile.js

```diff
@@ -41,6 +42,9 @@ gulp.task('test-server-stop', function () {
 gulp.task('mocha-test', function () {
     var stream = mochaPhantomJS();
     stream.write({path: 'http://localhost:21029/test/fixtures/index.html'});
+    stream.on('error', function(e) {
+        runSequence('test-server-stop', function () { throw e; });
+    });
```

Comment:
The timeout on travis (and was able to replicate locally) where if phantomjs fails out, the process hangs and after killing it the server is still running. This should handle it a bit more gracefully.